src/Avalonia.Base/Threading/DispatcherTimer.cs

```diff
@@ -210,7 +210,10 @@ public void Stop()
         /// </summary>
         private void InternalTick()
         {
-            Dispatcher.UIThread.InvokeAsync(_raiseTickAction, _priority);
+			if (IsEnabled) 
```

Comment:
This still doesn't look like the source of the problem. If `IsEnabled == false` then the timer has been stopped. So why is `InternalTick` being called when the timer has stopped?